core/controlplane/config/templates/cloud-config-controller

```diff
@@ -690,6 +690,8 @@ write_files:
         selector:
           matchLabels:
             k8s-app: calico-node
+        updateStrategy:
+          type: RollingUpdate
```

Comment:
Are we fully removing support for k8s 1.5? This will be rejected on k8s < 1.6 based on what I've seen before. It's not a problem for my use cases but some other users seems to want/need to use slightly older versions still.